utils.py

```diff
@@ -102,4 +102,40 @@ def get_model_coefficients(classifier, feature_set, covariate_names):
     coef_df['feature_set'] = feature_set
     
     
-    return coef_df
\ No newline at end of file
+    return coef_df
+
+def get_genes_coefficients(pca_object, classifier_object,
+                           expression_df):
+    """Identify gene coefficients from classifier after pca.
+
+    Args:
+        pca_object: The pca object from running pca on the expression_df.
+        classifier_object: The logistic regression classifier object.
+        expression_df: The original (pre-pca) expression data frame.
+
+    Returns:
+        gene_coefficients: A dataframe with entreze gene-ID, coefficient and
```

Comment:
Yes, gene names would be really helpful.

You can get them from https://github.com/cognoma/cancer-data/raw/1d961fb2201209b3ec4774aaadd36f36cf6d165e/data/expression-genes.tsv. You may even want to add the gene description in addition to symbol.